pkg/vcdsdk/gateway.go

```diff
@@ -1949,3 +1950,40 @@ func (gm *GatewayManager) UpdateLoadBalancer(ctx context.Context, lbPoolName str
 	}
 	return vsSummary.VirtualIpAddress, nil
 }
+
+func (gm *GatewayManager) FetchIpSpacesBackingGateway(ctx context.Context) ([]string, error) {
```

Comment:
Nit: a comment describing the function will be helpful. We have a small contract here that is not clear to me:
1. If there are no ipSpaceIds available, we will get a return of `nil, nil`. Is that a valid case?
2. Should we call function only when the gw is ip-space enabled? Else, do we get an error, or do we get `nil, nil` as above?